cypress/integration/brewery/CreateScenario-tests.spec.js

```diff
@@ -20,12 +20,19 @@ import { BreweryParameters } from '../../commons/actions/brewery';
 
 describe('Create scenario', () => {
   const randomString = utils.randomStr(7);
-  const scenarioMasterName = SCENARIO_NAME.SCENARIO_MASTER + randomString;
-  const scenarioChildName = SCENARIO_NAME.SCENARIO_CHILD + randomString;
-  const scenarioWithBasicTypesName = SCENARIO_NAME.SCENARIO_WITH_BASIC_TYPES + randomString;
-  const otherScenarioName = SCENARIO_NAME.OTHER_SCENARIO + randomString;
-  let scenarioMasterId, scenarioChildId, scenarioWithBasicTypesId, otherScenarioId;
-  let anotherScenarioUrlRegex;
+  const scenarioMasterName = utils.generateScenarioName(SCENARIO_NAME.SCENARIO_MASTER, randomString);
+  const scenarioChildName = utils.generateScenarioName(SCENARIO_NAME.SCENARIO_CHILD, randomString);
+  const scenarioWithBasicTypesName = utils.generateScenarioName(SCENARIO_NAME.SCENARIO_WITH_BASIC_TYPES, randomString);
+  const otherScenarioNumbers = 5;
+  const otherScenarioName = new Array(otherScenarioNumbers);
+  for (let i = 0; i < otherScenarioName.length; ++i) {
+    otherScenarioName[i] = utils.generateScenarioName(SCENARIO_NAME.OTHER_SCENARIO + i + ' - ', randomString);
+    console.log(otherScenarioName[i]);
+  }
+
+  let scenarioMasterId, scenarioChildId, scenarioWithBasicTypesId;
+  const otherScenarioId = new Array(otherScenarioNumbers);
+  const anotherScenarioUrlRegex = new Array(otherScenarioNumbers);
```

Comment:
Do we really need an array here ? It seems only the first value of the list is used in the tests